api/tests/certification/course/integration/infrastructure/repositories/sco-certification-candidate-repository_test.js

```diff
@@ -30,27 +40,43 @@ describe('Certification | Course | Integration | Repository | SCOCertificationCa
 
     it('retrieves the non disabled candidates that belong to the organization and division', async function () {
       // given
-      const sessionId = databaseBuilder.factory.buildSession().id;
+      const sessionId = databaseBuilder.factory.buildSession({ publishedAt: new Date('2024-01-01') }).id;
       const anOrganizationId = databaseBuilder.factory.buildOrganization().id;
       const nonDisabledOrganizationLearnerId = databaseBuilder.factory.buildOrganizationLearner({
         organizationId: anOrganizationId,
         division: '3ème A',
         isDisabled: false,
       }).id;
-      const nonDisabledCandidateId = databaseBuilder.factory.buildCertificationCandidate({
+      const nonDisabledCandidate = databaseBuilder.factory.buildCertificationCandidate({
```

Comment:
```suggestion
      const activeCandidate = databaseBuilder.factory.buildCertificationCandidate({
```
Qu'en pensez-vous ? Ça parait plus simple, mais je sais pas si ça porte le même niveau d'information 🤔

Reply:
j'y ai pensé aussi mais active ça peut paraître flou. 

Active donc pas deleted ? comme ce que propose la vue view-active-organization-learners
Active donc pas disabled ? 

Du coup je me range sur le nonDisabled pour être sûr :)

Reply:
Moi ça me va de pas changer :D